_posts/2018-06-27-inception-pipelines-pt4.markdown

```diff
@@ -0,0 +1,171 @@
+
+#### Audit AWS Account
+
+This account is where all things security and audit-ty related go. The account-level Inception Pipeline would define three IAM roles:
+
+* A heavily controlled **Administrator** account. Once defined, it would only need to be used in the case of emergency to perform actions (most likely in the AWS Console itself).
+* A more open role that can be assumed to make changes via the other pipelines. Called something like **SecurityOwner**?
+* Finally, a read-only role that can be given to developers, auditors, etc to view logs and other files stored in this account, and possibly called something like **SecurityReviewer**.
+
+Depending on your requirements, you will then have one (or more) pipelines which define things like:
+
+* the S3 bucket(s) for CloudTrail (in your other accounts) to store events.
+* Definitions for data retention.
+* CloudWatch Alarms for CloudTrail events.
+
+#### Shared Services AWS Account
+
+Your shared services account defines all the aspects that your other accounts consume but are not allowed to manage. For example, you'll have the standard account-level Inception Pipeline to managed IAM roles. You might then, as an organisation, want to centralise the management of Route 53. So, you would create another pipeline to manage the creation of Route 53 hosted zones and records sets.
+
+Other shared service ideas include:
+
+* [Direct Connect](https://aws.amazon.com/directconnect/)
+* [Service Catalog](https://aws.amazon.com/servicecatalog/)
+* [VPC Endpoint Services (AWS PrivateLink)](https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/endpoint-service.html)
+* [API Gateway Private Integrations](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-private-integration.html)
+
+The last two points make it easier and safer for a service from the Shared Service account to be consumed by VPC's in other accounts through VPC Peering.
+
+#### Sandbox AWS Account
+
+The Sandbox AWS Account is used by developers as a safe place to try out all the services that AWS provides. It is a volatile landscape with lots of things being created and destroyed all the time. Developers should have full organisational and AWS permissions here to experiment. To help prevent cost blowouts, AWS Budgets should be used to alert when the forecasted, and actual, spend hits predefined levels.
+
+It is also possible to define a Lambda in the root account that periodically deletes the Sandbox account and recreates it in a clean state.
+
+The account Inception Pipeline defines the roles that can be assumed, as well as setup AWS Budget alerts. When people are experimenting, they would have their own copy of the pipeline.
+
+### Managing Aspects
+
+Now we move into the interesting bits and answer the question "How do I structure my accounts\pipeline for my application?".
```

Comment:
some places you use a forward slash others a backslash. Not sure if intentional?